optimizer/lib/HtmlDomHelper.js

```diff
@@ -31,7 +31,21 @@ function findMetaViewport(head) {
   return null;
 }
 
+/**
+   * Skips the subtree that is descending from the current node.
+   * @param {Node} node the node that has its subtree being skipped
+   * @return {Node} the appropriate "next" node that will skip the current
+   * subtree.
+   */
+function nextNode(node) {
```

Comment:
Let's call this `skipNodeAndChildren()` to disambiguate with the instance method `node.nextNode()`.